src/renderer/frames/cluster-frame/init-cluster-frame/init-cluster-frame.ts

```diff
@@ -84,7 +83,7 @@ export const initClusterFrame =
           `${logPrefix} Unload dashboard, clusterId=${(hostedCluster.id)}, frameId=${frameRoutingId}`,
         );
 
-        unmountComponentAtNode(rootElem);
+        requestClusterDisconnection(hostedCluster.id);
```

Comment:
Unmount was here on purpose, without unmount Lens might leak watch etc requests because they happen within nodejs runtime (not within browser runtime).

Reply:
I will find another way to fix